src/Root.js

```diff
@@ -12,6 +12,7 @@ import CONFIG from './constants/config'
 import ROUTE from './constants/route'
 import SignIn from './components/auth/SignIn'
 import SignUp from './components/auth/SignUp'
+import TRANSACTIONS from './components/transactions'
```

Comment:
Only constants should be capitalized

Reply:
We should start naming full screen components as scenes.
i.e. `TransactionScene`, `SignUpScene` etc